eventsdiscovery/src/main/java/com/schedjoules/client/eventsdiscovery/EventsDiscovery.java

```diff
@@ -84,4 +86,9 @@
      * @return
      */
     EventsDiscovery withStartAtOrAfter(DateTime startAtOrAfter);
+
+    /**
+     * Sets the event categories filter
+     */
+    EventsDiscovery withCategories(Iterable<Uri> categories);
```

Comment:
Ok, that makes sense, but I wonder if `Category` is the right name for the model then. Anyway, let's keep this for now.